modules/hello-world/pages/start-using-sdk.adoc

```diff
@@ -146,25 +137,84 @@ $ echo 'export PATH="/usr/local/bin:"$PATH' >> ~/.bash_profile
 $ brew install openssl
 ----
 
-.Installing the SDK
+
+=== Microsoft Windows
+
+Download and install Python from https://www.python.org/downloads[python.org^].  Best practice is to use a Python virtual environment such as _venv_ or _pyenv_.
+
+TIP: Checkout the https://github.com/pyenv-win/pyenv-win[pyenv-win^] project to manage multiple versions of Python.
+
+Wheels are available on Windows for Python 3.7, 3.8 and 3.9.
+
+
+== Installing on Linux
+
+First, make sure the <<linux, requirements>> have been installed.
+
+Install the latest Python SDK:
+
+[source,console]
+----
+$ python3 -m pip install couchbase
+----
+
+== Installation on Mac OS
+
+First, make sure the <<mac-os, requirements>> have been installed.
+
+Install the latest Python SDK:
+
 [source,console]
 ----
 $ sudo -H python3 -m pip install couchbase
 ----
-====
 
 
 == Installing on Microsoft Windows
 
+First, make sure the <<microsoft-windows, requirements>> have been installed.
+
+NOTE: Commands assume user is working within a virtual environment.
+
+Install the latest Python SDK (if using Python 3.7, 3.8 or 3.9):
+
 [source,console]
 ----
-$ python3 -m pip install couchbase
+(python39) C:\Users\user1\PythonProjects>python -m pip install couchbase
```

Comment:
Should (python39) be outside this copypastable command?

Reply:
Yes, the command to copy/paste should just be python -m pip install couchbase.  I suppose since I have the NOTE above stating the use of a virtual environment, we can just change that to python3 -m pip install couchbase.